resources/views/layouts/app.blade.php

```diff
@@ -55,51 +55,59 @@
 
-                                    <div class="text-lg hover:text-gray-400">{{ Auth::user()->name }}</div>
+                                <div class="text-lg hover:text-gray-400">{{ Auth::user()->name }}</div>
 
-                                    <div class="ml-1">
-                                        <svg class="fill-current h-4 w-4" xmlns="http://www.w3.org/2000/svg"
-                                            viewBox="0 0 20 20">
-                                            <path fill-rule="evenodd"
-                                                d="M5.293 7.293a1 1 0 011.414 0L10 10.586l3.293-3.293a1 1 0 111.414 1.414l-4 4a1 1 0 01-1.414 0l-4-4a1 1 0 010-1.414z"
-                                                clip-rule="evenodd" />
-                                        </svg>
-                                    </div>
-                                </button>
-                            </li>
-                        </x-slot>
+                                <div class="ml-1">
+                                    <svg class="fill-current h-4 w-4" xmlns="http://www.w3.org/2000/svg"
+                                        viewBox="0 0 20 20">
+                                        <path fill-rule="evenodd"
+                                            d="M5.293 7.293a1 1 0 011.414 0L10 10.586l3.293-3.293a1 1 0 111.414 1.414l-4 4a1 1 0 01-1.414 0l-4-4a1 1 0 010-1.414z"
+                                            clip-rule="evenodd" />
+                                    </svg>
+                                </div>
+                            </button>
+                        </li>
+                    </x-slot>
 
-                        <x-slot name="content">
-                            <!-- Authentication -->
-                            <form method="GET" action="{{ route('dashboard') }}">
-                                <x-dropdown-link :href="route('dashboard')" onclick="event.preventDefault();
+                    <x-slot name="content">
+                        <!-- Authentication -->
+                        <form method="GET" action="{{ route('dashboard') }}">
+                            <x-dropdown-link :href="route('dashboard')" onclick="event.preventDefault();
                                                 this.closest('form').submit();">
-                                    {{ __('Profile') }}
-                                </x-dropdown-link>
-                            </form>
-                            <form method="POST" action="{{ route('logout') }}">
-                                @csrf
-                                <x-dropdown-link :href="route('logout')" onclick="event.preventDefault();
+                                {{ __('Profile') }}
+                            </x-dropdown-link>
+                        </form>
+                        @if(auth()->user()->type == 2)
```

Comment:
Accounts with type 3 can't view the link to users dashboard. Do we have a type 3 user account or is it an error in the seeding of the database?